jobs/JGLOBAL_SNOW_ENSEMBLE_ANALYSIS

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export DATA=${DATA:-${DATAROOT}/${RUN}esnowanl_${cyc}}
+source "${HOMEgfs}/ush/jjob_header.sh" -e "esnowanl" -c "base esnowanl"
+
+##############################################
+# Set variables used in the script
+##############################################
+# Ignore possible spelling error (nothing is misspelled)
+# shellcheck disable=SC2153
+GDUMP="gdas"
+GDUMP_ENS="enkf${GDUMP}"
+export GDUMP
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COMIN_OBS COMOUT_ATMOS_ANALYSIS COMOUT_CONF
+MEMDIR="ensstat" YMD=${PDY} HH=${cyc} declare_from_tmpl COMOUT_SNOW_ANALYSIS
+
+mkdir -m 775 -p "${COMOUT_SNOW_ANALYSIS}" "${COMOUT_CONF}"
```

Comment:
```suggestion
mkdir -p "${COMOUT_SNOW_ANALYSIS}" "${COMOUT_CONF}"
```
`-m 775` gives group write permissions to the directory.  Don't think we want that.

Reply:
I forget where I pulled this from, other jobs definitely do this, is there a reason why you think?

Reply:
yeah, we noticed it was done in the wave jobs.
I looked at the permissions, and it gives write permission to members belonging to the group.  I don't think that is right.  
ops runs in the ops/spa role account, so they don't need this.